tuxemon/menu/menu.py

```diff
@@ -690,15 +709,14 @@ def change_selection(self, index: int, animate: bool = True) -> None:
         """
         previous = self.get_selected_item()
         if previous is not None:
-            previous.in_focus = False  # clear the focus flag of old item, if any
-        self.selected_index = index  # update the selection index
-        self.menu_select_sound.play()  # play a sound
-        # move cursor and [maybe] animate it
+            previous.in_focus = False
```

Comment:
yes, because the code seems obvious what they do.  the alternative is the keep them but move them to the line above the code.  black will do an ungly thing to inline comments, so i've been moving them to the line before, or removeing them if the code seems fine without it.  if you disagree that the comments are not redundant, i can add them back
```python
y = (
    (m * x) + b
) # this is an ugly comment regarding 'slope-intercept formula'
```

Reply:
No, there are redundant indeed. I was only worried that they were removed automatically.